src/frontend/src/lib/components/hero/Balance.svelte

```diff
@@ -1,10 +1,15 @@
 <script lang="ts">
-	import { isNullish, nonNullish } from '@dfinity/utils';
+	import { nonNullish } from '@dfinity/utils';
+	import { getContext } from 'svelte';
 	import TokenExchangeBalance from '$lib/components/tokens/TokenExchangeBalance.svelte';
 	import Amount from '$lib/components/ui/Amount.svelte';
+	import { HERO_CONTEXT_KEY, type HeroContext } from '$lib/stores/hero.store';
 	import type { OptionTokenUi } from '$lib/types/token';
 
 	export let token: OptionTokenUi;
+	``;
```

Comment:
What's this line?

Reply:
not really sure where it comes from, probably a "fat finger" from me

deleted